app/views/orders/new.html.haml

```diff
@@ -0,0 +1,15 @@
+= form_tag(menu_orders_path(@menu)) do
```

Comment:
I'd recommend to use `simple_form`. you will hardly find a project without simple_form anyway))